### ISSUE TITLE
Could not reduce power via /power -1 on a card in play 11/26/16

### ISSUE THREAD

**Issue by XMaratX** [cryogen/throneteki#73]

**Comment by cryogen** [comment]
Not sure what you mean by this, isn't negative power an illegal game state?  If you merely intended to reduce power by 1 just do /power <correctpowervalue>

**Comment by XMaratX** [comment]
The situation involved a military challenge in which I informed my opponent that I was activating Sworn Brother (recognizing that it is not implemented, so we did it via the chat) as part of a block. One of the attackers was Ser Jorah Mormont. The game resolved the challenge with Mormont's controller winning (thus activating his Renown) though in reality I won as defender. We didn't see anyway to remove the power token on Ser Jorah using the /power action (I had hoped that the command /power -1 would remove a power token from him but evidently numbers after a command are ignored). /power only added a power token.
